main.go

```diff
@@ -26,84 +24,74 @@ var (
 	env utils.EnvConfig
 )
 
-func main() {
-	logger := keptncommon.NewLogger("", "", utils.ServiceName)
-
-	env = utils.EnvConfig{}
-
-	if err := envconfig.Process("", &env); err != nil {
-		logger.Error(fmt.Sprintf("Failed to process env var: %s", err))
-	}
-
-	logger.Debug(fmt.Sprintf("Configuration service: %s", env.ConfigurationServiceURL))
-	logger.Debug(fmt.Sprintf("Port: %d, Path: %s", env.Port, env.Path))
+const serviceName = "prometheus-service"
+const envVarLogLevel = "LOG_LEVEL"
+const monitoringTriggeredEvent = "sh.keptn.event.configure-monitoring.triggered"
+const getSliTriggeredEvent = "sh.keptn.event.get-sli.triggered"
 
-	// start internal CloudEvents handler (on port env.Port)
-	os.Exit(_main(env))
-}
-
-func _main(env utils.EnvConfig) int {
-	ctx := context.Background()
-	ctx = cloudevents.WithEncodingStructured(ctx)
-
-	p, err := cloudevents.NewHTTP()
-	if err != nil {
-		log.Fatalf("failed to create protocol: %s", err.Error())
+func main() {
+	if os.Getenv(envVarLogLevel) != "" {
+		logLevel, err := logrus.ParseLevel(os.Getenv(envVarLogLevel))
+		if err != nil {
+			logrus.WithError(err).Error("could not parse log level provided by 'LOG_LEVEL' env var")
+			logrus.SetLevel(logrus.InfoLevel)
+		} else {
+			logrus.SetLevel(logLevel)
+		}
 	}
 
-	ceHandler, err := cloudevents.NewHTTPReceiveHandler(ctx, p, gotEvent)
-	if err != nil {
-		log.Fatalf("failed to create handler: %s", err.Error())
-	}
+	log.Printf("Starting %s", serviceName)
 
 	http.HandleFunc("/", HTTPGetHandler)
-	http.Handle(env.Path, ceHandler)
-	err = http.ListenAndServe(":8080", nil)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	return 0
-}
+	go func() {
+		err := http.ListenAndServe(":8080", nil)
+		if err != nil {
+			log.Fatalf("Error with HTTP server: %e", err)
+		}
+	}()
```

Comment:
Please add a comment what we are doing here, and why, e.g., "Creating an HTTP Listener on port 8080 to receive Alerts from Prometheus directly"